app/javascript/components/server-components/Admin/UserStats.tsx

```
@@ -0,0 +1,38 @@
+        setUserStats(await response.text());
+      } catch (e) {
+        assertResponseError(e);
+        showAlert(e.message, "error");
+      }
+    };
+
+    void fetchUserStats();
+  });
+
+  return userStats ? (
+    <ul className="inline" dangerouslySetInnerHTML={{ __html: userStats }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)